apps/dialtone-documentation/docs/.vuepress/client.js

```diff
@@ -18,6 +18,18 @@ import ComponentCombinator from './baseComponents/ComponentCombinator.vue';
 import ExampleTabs from './baseComponents/ExampleTabs.vue';
 import SvgLoader from './baseComponents/SvgLoader.vue';
 
+// Illustrations
+import SpotPublish from '@dialpad/dialtone/lib/dist/vue/spot/SpotPublish.vue';
+import SpotWirelessScreenshare from '@dialpad/dialtone/lib/dist/vue/spot/SpotWirelessScreenshare.vue';
+import SpotFemaleLaptopTyping from '@dialpad/dialtone/lib/dist/vue/spot/SpotFemaleLaptopTyping.vue';
+import SpotMaleLaptopTyping from '@dialpad/dialtone/lib/dist/vue/spot/SpotMaleLaptopTyping.vue';
+import SpotFileUpload from '@dialpad/dialtone/lib/dist/vue/spot/SpotFileUpload.vue';
+import SpotBrowserTableGraph from '@dialpad/dialtone/lib/dist/vue/spot/SpotBrowserTableGraph.vue';
+import SpotBrowserListCallout from '@dialpad/dialtone/lib/dist/vue/spot/SpotBrowserListCallout.vue';
+import SpotMind from '@dialpad/dialtone/lib/dist/vue/spot/SpotMind.vue';
+import SpotBlankSpace from '@dialpad/dialtone/lib/dist/vue/spot/SpotBlankSpace.vue';
+import SpotVectorVortex from '@dialpad/dialtone/lib/dist/vue/spot/SpotVectorVortex.vue';
```

Comment:
I'll see if I can figure out the problem with that svgloader component. Would be nice if we didn't have to do this.

Reply:
I suspect it has something to do with the defs but couldn't figure it out. Inspecting the svg, everything seems ok.